server/mount.js

```diff
@@ -0,0 +1,53 @@
+const {
+  assocPath, compose, curry, identity, is, path, pipe, tap, unless, when
+} = require('ramda')
+
+const cleanMeta     = require('./cleanMeta')
+const defaultLogger = require('./logger')
+const error         = require('./error')
+const formatErrors  = require('./formatErrors')
+
+// mount :: { k: v } -> (Socket, Function) -> ()
+const mount = (opts={}) => {
+  const {
+    app    = identity,
+    cry    = defaultLogger,
+    logger = defaultLogger
+  } = opts
+
+  const connected = (socket, done) => {
+    const { session } = socket.handshake.query
+
+    const addMeta = pipe(
+      unless(
+        path(['meta', 'session']),
```

Comment:
Falsy trap?

Reply:
Previous versions of `sox` included the `session` as a query param.  The new version already includes it in the `action.meta` client-side.  The goal was to be backwards compatible, so this accepts both.  Since it's all handled by the library, and not the user, I'm not too worried about it being falsy... falsely.